lib/executor/variable_looping_vus.go

```diff
@@ -614,7 +614,7 @@ func (vlv VariableLoopingVUs) Run(ctx context.Context, out chan<- stats.SampleCo
 	handleNewScheduledVUs := func(newScheduledVUs uint64) {
 		if newScheduledVUs > currentScheduledVUs {
 			for vuNum := currentScheduledVUs; vuNum < newScheduledVUs; vuNum++ {
-				vuHandles[vuNum].start()
+				_ = vuHandles[vuNum].start() // TODO handle error
```

Comment:
Would this involve a larger VLV refactor? Are we sure we can postpone it?

Reply:
we've been ignoring the errors up until now ... so ... I would say - yes, although if possible we should probably prioritise it